maintainers/maintainer-list.nix

```diff
@@ -3859,6 +3923,7 @@
   };
   frogamic = {
     email = "frogamic@protonmail.com";
+    matrix = "@frogamic:matrix.org";
```

Comment:
Mine is correct however I don't use or check Matrix regularly so i would not publicise it.

Reply:
I've removed it, now. Thank you.